sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/GoneAndRetryPolicyWithSpyClientTest.java

```diff
@@ -69,7 +69,8 @@ static Builder createDCBuilder(Protocol protocol) {
             .withConnectionPolicy(connectionPolicy)
             .withConsistencyLevel(ConsistencyLevel.SESSION)
             .withContentResponseOnWriteEnabled(true)
-            .withMasterKeyOrResourceToken(TestConfigurations.MASTER_KEY);
+            .withMasterKeyOrResourceToken(TestConfigurations.MASTER_KEY)
+            .withClientTelemetryConfig(new ClientTelemetryConfig(false));
```

Comment:
This should not be necessary, correct? Because right now ClientTelemetry is disabled by default - if we enable it by default - we would probably want to enable it in most unit tests as well?

Saw this is in a couple of other places as well? I would simply revert the changes wherever you set "withClientTelemetryConfig(false)" except if it is a test checking that actually no telemetry is emitted etc.

Reply:
it is kind necessary because currently I have a check ClientTelemetryConfig can not be null in clientTelemetry. 
I can relax the requirement, but by theory, it should never happen. 

And here we are directly using Build() (which is internal) to start the initialize, not the public interface, that is why I have the specific setting. 

hmm, and for unit tests, do we want to send telemetry to client telemetry?

Reply:
Discussed offline, it makes sense especially from point of view that our unit tests should do by default what customer will get by default.

And my original concern is also about keep default value in two places, but there is also some over thinking, and there is a way to avoid it. 

so I have reverted them back.